world/models.py

```diff
@@ -68,13 +70,49 @@ class Location(SingletonModel):
         default="http://www.fema.gov",
         help_text="A link to your local office of emergency management."
     )
-    
+
     def __unicode__(self):
         return u"Location Information"
 
+    @staticmethod
+    def get_data_bounds():
+        bounds = {
+
+    ######################################################
+    # GENERATED CODE GOES HERE
+    # DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
+    # Location
+    # END OF GENERATED CODE BLOCK
+    ######################################################
+        }
+
+        # The smallest/largest possible values, as appropriate, so the map will display
+        # something if there is no data
+        north = [-180]
+        west = [-90]
+        south = [180]
+        east = [90]
```

Comment:
Two things here:
1) Are you sure you have lat & lon the right way round?  N & S should only go to +/-90, while E & W are +/-180.
2) Non-obvious geo trick: because the Mercator projection turns the width of every line of latitude into the width of the equator, it typically runs into div/0 problems at the poles (since they are singular points with no width at all).  So it's safer to cut N/S bounds short of the poles themselves.  I usually just use +/-80, since Mercator is so horrible for high latitudes that anyone who needs to map those areas will need to make a bunch of other changes anyway.